src/UhdmAst.cpp

```diff
@@ -208,6 +208,42 @@ FileLine* make_fileline(vpiHandle obj_h) {
     return fl;
 }
 
+AstNodeDType* apply_packed_ranges(FileLine* fl, vpiHandle obj_h, AstNodeDType* dtypep,
```

Comment:
Since this is essentially a helper function (wrapping an already made AST node), I'd go with Verilator naming convention (same for `apply_unpacked_ranges`).

Reply:
Done